cmd/juju/charmhub/download_test.go

```diff
@@ -136,9 +136,32 @@ func (s *downloadSuite) TestRunWithCustomCharmHubURL(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 }
 
-func (s *downloadSuite) TestRunWithUnsupportedSeries(c *gc.C) {
+func (s *downloadSuite) TestRunWithUnsupportedSeriesPicksFirstSuggestion(c *gc.C) {
 	defer s.setUpMocks(c).Finish()
 
+	url := "http://example.org/"
+
+	s.expectRefreshUnsupportedSeries()
+	s.expectRefresh(url)
+	s.expectDownload(c, url)
+	s.expectFilesystem(c)
+
+	command := &downloadCommand{
+		charmHubCommand: s.newCharmHubCommand(),
+	}
+	command.SetFilesystem(s.filesystem)
+	err := cmdtesting.InitCommand(command, []string{"test"})
+	c.Assert(err, jc.ErrorIsNil)
+
+	ctx := commandContextForTest(c)
+	err = command.Run(ctx)
+	c.Assert(err, jc.ErrorIsNil)
+}
+
+func (s *downloadSuite) TestRunWithUnsupportedSeriesReturnsSecondAttempt(c *gc.C) {
+	defer s.setUpMocks(c).Finish()
+
+	s.expectRefreshUnsupportedSeries()
```

Comment:
@SimonRichardson this merged test fails on
```
c.Assert(err, gc.ErrorMatches, `"test" does not support series "jammy" in channel "stable".  Supported series are: bionic, trusty, xenial.`)
```

I'm not sure why we are returning a different error now.

```
13:56:45 FAIL: download_test.go:161: downloadSuite.TestRunWithUnsupportedSeriesReturnsSecondAttempt
13:56:45 
13:56:45 download_test.go:176:
13:56:45     c.Assert(err, gc.ErrorMatches, `"test" does not support series "jammy" in channel "stable".  Supported series are: bionic, trusty, xenial.`)
13:56:45 ... error string = "\"test\" does not support series \"bionic\" in channel \"stable\".  Supported series are: bionic, trusty, xenial."
13:56:45 ... regex string = "\"test\" does not support series \"jammy\" in channel \"stable\".  Supported series are: bionic, trusty, xenial."
```